i8_terminal/commands/metrics/metrics_historical.py

```diff
@@ -149,6 +153,45 @@ def create_fig(
     return fig
 
 
+def historical_metrics_df2tree(df: DataFrame) -> Tree:
+    df = df.pivot(index=["Metric", "Ticker"], columns="Period", values="value").reset_index()
+    reversed_periods = [reverse_period(p) for p in list(df.columns[2:])]
+    reversed_periods.sort(reverse=True)
+    sorted_periods = [reverse_period(rp) for rp in reversed_periods]
+    col_width = 15
+    plot_title = f"Historical {' and '.join(list(set(df['Metric'])))}"
```

Comment:
`A, B, and C historical metrics`. A, B, C are the tickers. No need to have the name of metrics in the title.